EpikCord/client/client.py

```diff
@@ -34,21 +35,12 @@ def __init__(
         self.guilds: GuildManager = GuildManager(self)
         self.channels: ChannelManager = ChannelManager(self)
         self.presence: Presence = Presence(status=status, activity=activity)
-        self._components = {}
+        self._components: Dict[str, Coroutine] = {}
```

Comment:
Wrong typing? If it maps string to coroutine functions then it should be `Callable[..., Coroutine[Any, Any, Any]]`

Reply:
That is correct, I will fix this.